src/Composer/Command/DisableOriginalDrupalKernelSubstitutionCommand.php

```diff
@@ -21,16 +21,23 @@
 class DisableOriginalDrupalKernelSubstitutionCommand extends BaseCommand
 {
     /**
-     * @var string
+     * @const string
```

Comment:
`@const` does not seem to be a valid phpdoc tag. cf https://stackoverflow.com/questions/6706051/what-is-the-correct-way-to-write-phpdocs-for-constants (not the valid answer ^^)

Reply:
Changed.
I had seen this on Drupal or zend-diactoros package and never checked.